keras/objectives.py

```diff
@@ -11,6 +11,10 @@ def mean_absolute_error(y_true, y_pred):
     return K.mean(K.abs(y_pred - y_true), axis=-1)
 
 
+def mean_identity_error(y_true, y_pred):
+    return K.mean(y_pred, axis=-1)
```

Comment:
Thanks for the insight! I will consider adding in a trivial usage of the y_true labels to suppress the Theano warnings.